src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -311,6 +314,10 @@ private function updateAddressComponent(AddressBuilder $builder, $type, $values)
                 $builder->setSubLocality($values->long_name);
                 break;
 
+            case 'subpremise':
+                $builder->subpremise = $values->long_name;
```

Comment:
Not sure we can do this. `$builder->subpremise` is not public. One should aways use setters.